### ISSUE TITLE
Unable to preventDefault inside passive event listener

### ISSUE THREAD

**Issue by aidant** [acidb/mobiscroll#511]
After upgrading to `@mobiscroll/angular@5.21.2` from `@mobiscroll/angular@5.20.0` we have started seeing the following error:

```
PreventDefaultPassive: intervention: Unable to preventDefault inside passive event listener due to target being treated as passive. See https://www.chromestatus.com/feature/6662647093133312
	node_modules/@mobiscroll/angular/dist/esm5/mobiscroll.angular.min.js at line 2:319828
```

The error occurs when scrolling vertically on the time section of the `mbsc-datepicker` component. We are only seeing this issue in Chrome and Edge versions `109` and above (note `109` was the latest when we switched to `5.21.2`).

**Comment by dioslaska** [comment]
So far I could not reproduce this. Could you share the exact code you're using and the exact angular versions (output of `ng version`)?

**Comment by aidant** [comment]
To make this confusing, we've noticed this error only occurs if we have previously navigated to a page in our SPA that contains a map and we are still observing the issue in mobiscroll version `5.20.0`. If we revert the Google Maps JavaScript API down to version `3.50` we are unable to reproduce the issue which leads us to be suspicious of the [Google Maps JavaScript API version `3.51.6`](https://developers.google.com/maps/documentation/javascript/releases#3.51.6) as this was released at the same time as this issue was first observed.

I don't have a code example I can share but I might be able to make one if needed. Our Angular version is `15.1.1`.

I've also opened an issue with the [Google Maps JavaScript API](https://issuetracker.google.com/issues/269959725).
